static-builder-gnu.Dockerfile

```diff
@@ -112,8 +112,9 @@ ENV SPC_LIBC='glibc'
 ENV SPC_CMD_VAR_PHP_MAKE_EXTRA_LDFLAGS_PROGRAM='-Wl,-O3 -pie'
 ENV SPC_CMD_VAR_PHP_MAKE_EXTRA_LIBS='-ldl -lpthread -lm -lresolv -lutil -lrt'
 ENV SPC_OPT_DOWNLOAD_ARGS='--ignore-cache-sources=php-src'
-ENV SPC_OPT_BUILD_ARGS=''
+ENV SPC_OPT_BUILD_ARGS='--with-config-file-path=/etc/frankenphp/php.ini --with-config-file-scan-dir=/etc/frankenphp/php.d'
 ENV SPC_REL_TYPE='binary'
+ENV EXTENSION_DIR='/usr/lib64/frankenphp/modules'
```

Comment:
so this one is a doozy, because the default paths are:

```
; Ubuntu/Debian: /usr/lib/php/{PHP_VERSION}/
; RHEL: /usr/lib64/php/modules
; Alpine: /usr/lib/php{PHP_VERSION}/modules
; where {PHP_VERSION} is 84 for php 8.4
```

I cannot think of a way to build these for different paths without rebuilding and I also cannot think of a single path that we could use on every distro. Maybe `/usr/bin/frankenphp-ext`? Most definitely bad practise...

Reply:
Wait, `/usr/lib/frankenphp/modules` should be fine on each distribution. It may be slightly off best-practise for RHEL since extensions would be 64 bit dll's and would typically go into /usr/lib64, but this would keep the location unified.